devicetypes/smartthings/zigbee-battery-accessory-dimmer.src/zigbee-battery-accessory-dimmer.groovy

```diff
@@ -22,8 +22,9 @@ metadata {
 		capability "Health Check"
 		capability "Switch"
 		capability "Switch Level"
-		
+
 		fingerprint profileId: "0104", inClusters: "0000,0001,0003,0020,FC11", outClusters: "0003,0004,0006,0008,FC10", manufacturer: "sengled", model: "E1E-G7F", deviceJoinName: "Sengled Smart Switch"
+		fingerprint profileId: "0104", inClusters: "0000, 0001, 0003, 0020, 0B05", outClusters: "0003, 0006, 0008, 0019", manufacturer: "Centralite Systems", model: "3131-G", deviceJoinName: "Centralite Smart Switch"
```

Comment:
spacing between clusters should be omitted

Reply:
Sure